From 0332c009bd0581ab9a75a4ea80af92bb2d6b8b1f Mon Sep 17 00:00:00 2001
From: Dan Walsh <dwalsh@redhat.com>
Date: Tue, 20 Sep 2011 15:46:38 -0400
Subject: [PATCH 02/11] Fix sepol calls to work with latest libsepol

---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 577ce48..2a5b55b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -521,7 +521,7 @@ else
                   [AC_LANG_SOURCE([
 #include <sepol/policydb/expand.h>
 int main () {
-  return role_set_expand(NULL, NULL, NULL, NULL);
+  return role_set_expand(NULL, NULL, NULL, NULL, NULL);
 }])],
                   sepol_new_user_role_mapping="yes",
                   sepol_new_user_role_mapping="no")
@@ -578,7 +578,7 @@ if test ${sepol_check_boolmap} = "yes"; then
                          [AC_LANG_SOURCE([
 #include <sepol/policydb/expand.h>
 int main () {
-  return expand_module_avrules(NULL, NULL, NULL, NULL, NULL, 0, 0);
+  return expand_module_avrules(NULL, NULL, NULL, NULL, NULL, 0, 0, 0, 0);
 }])],
                          AC_MSG_RESULT([yes]),
                          AC_MSG_ERROR([this version of libsepol is incompatible with SETools]))
-- 
1.8.5.3

