diff -up setools-3.3.7/Makefile.am.noship setools-3.3.7/Makefile.am
--- setools-3.3.7/Makefile.am.noship	2008-02-22 14:06:28.000000000 -0500
+++ setools-3.3.7/Makefile.am	2013-11-27 09:30:01.467627063 -0500
@@ -78,9 +78,9 @@ install-logwatch:
 	$(MAKE) -C $(top_srcdir)/seaudit install-logwatch
 
 .PHONY: libqpol libapol libpoldiff libsefs libseaudit \
-	apol secmds seaudit sediff sediffx sechecker \
+	apol secmds seaudit sediff sechecker\
 	install-logwatch help \
-	seinfo sesearch indexcon findcon replcon searchcon \
+	seinfo sesearch findcon \
 	packages
 
 seinfo: libqpol libapol
diff -up setools-3.3.7/configure.ac.noship setools-3.3.7/configure.ac
--- setools-3.3.7/configure.ac.noship	2013-11-27 09:27:14.279196042 -0500
+++ setools-3.3.7/configure.ac	2013-11-27 09:27:14.309196384 -0500
@@ -63,7 +63,7 @@ if test ${ac_cv_prog_cc_c99} = "no"; the
 fi
 AC_PROG_CXX
 AC_LANG([C])
-AC_PROG_LIBTOOL
+AC_PROG_RANLIB
 AC_PROG_LN_S
 AC_PROG_LEX
 AC_PROG_YACC
@@ -194,7 +194,7 @@ AC_ARG_ENABLE(swig-java,
               enable_jswig="$enableval")
 if test "x${enable_jswig}" = xyes; then
    if test ${do_swigify} = no; then
-      AC_PROG_SWIG(1.3.28)
+      AC_PROG_SWIG(2.0.0)
    fi
    AC_JAVA_OPTIONS
    if test "x$JAVAPREFIX" = x; then
@@ -222,7 +222,7 @@ AC_ARG_ENABLE(swig-python,
               enable_pyswig="$enableval")
 if test "x${enable_pyswig}" = xyes; then
    if test ${do_swigify} = no; then
-      AC_PROG_SWIG(1.3.28)
+      AC_PROG_SWIG(2.0.0)
    fi
    AM_PATH_PYTHON(2.3)
    SWIG_PYTHON
@@ -230,7 +230,7 @@ if test "x${enable_pyswig}" = xyes; then
    do_swigify=yes
 fi
 if test ${do_swigify} = "yes"; then
-   AC_PROG_SWIG(1.3.28)
+   AC_PROG_SWIG(2.0.0)
 fi
 build_apol=yes
 AC_ARG_ENABLE(swig-tcl,
@@ -239,7 +239,7 @@ AC_ARG_ENABLE(swig-tcl,
               enable_tclswig="$enableval", enable_tclswig="yes")
 if test "x${enable_tclswig}" = xyes; then
    if test ${do_swigify} = no; then
-      AC_PROG_SWIG(1.3.28)
+      AC_PROG_SWIG(2.0.0)
    fi
    TEA_INIT(3.5)
    TEA_PATH_TCLCONFIG
diff -up setools-3.3.7/man/Makefile.am.noship setools-3.3.7/man/Makefile.am
--- setools-3.3.7/man/Makefile.am.noship	2007-08-02 17:16:33.000000000 -0400
+++ setools-3.3.7/man/Makefile.am	2013-11-27 09:27:14.306196350 -0500
@@ -1,19 +1,10 @@
 if BUILD_GUI
   MAYBEMANS = apol.1 \
-	seaudit.8 seaudit-report.8 \
-	sediffx.1
+	seaudit.8
 endif
 
 EXTRA_DIST=$(man_MANS) apol.1 \
-	seaudit.8 seaudit-report.8.in \
-	sediffx.1
+	seaudit.8
 
-man_MANS = findcon.1 indexcon.1 replcon.1 \
-	sechecker.1 \
-	sediff.1 \
+man_MANS = findcon.1 sediff.1 sechecker.1 \
 	seinfo.1 sesearch.1 $(MAYBEMANS)
-
-seaudit-report.8: seaudit-report.8.in Makefile
-	sed -e 's|\@setoolsdir\@|$(setoolsdir)|g' $< > $@
-
-CLEANFILES = seaudit-report.8
diff -up setools-3.3.7/seaudit/Makefile.am.noship setools-3.3.7/seaudit/Makefile.am
--- setools-3.3.7/seaudit/Makefile.am.noship	2008-02-22 14:06:28.000000000 -0500
+++ setools-3.3.7/seaudit/Makefile.am	2013-11-27 09:27:14.306196350 -0500
@@ -1,5 +1,4 @@
 setoolsdir = @setoolsdir@
-bin_PROGRAMS = seaudit-report
 sbin_PROGRAMS = seaudit
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
@@ -20,13 +19,10 @@ LDADD = @SELINUX_LIB_FLAG@ @SEAUDIT_LIB_
 dist_setools_DATA = \
 	seaudit.glade \
 	seaudit_help.txt \
-	seaudit-report.conf \
-	seaudit-report.css \
 	seaudit.png seaudit-small.png
 
 nodist_setools_DATA = \
-	dot_seaudit \
-	seaudit-report-service
+	dot_seaudit
 
 seaudit_SOURCES = \
 	filter_view.c filter_view.h \
@@ -50,31 +46,12 @@ seaudit_DEPENDENCIES = $(top_builddir)/l
 dot_seaudit: dot_seaudit.in Makefile
 	sed -e 's|\@setoolsdir\@|$(setoolsdir)|g' $< > $@
 
-seaudit_report_SOURCES = seaudit-report.c
-seaudit_report_DEPENDENCIES = $(top_builddir)/libseaudit/src/libseaudit.so \
-                              $(top_builddir)/libapol/src/libapol.so \
-                              $(top_builddir)/libqpol/src/libqpol.so
-
 logwatch = $(DESTDIR)/etc/logwatch
 LOGWATCH_GROUP = $(logwatch)/conf/logfiles
 LOGWATCH_SERVICE = $(logwatch)/conf/services
 LOGWATCH_FILTER = $(logwatch)/scripts/services
 
-dist_noinst_DATA = dot_seaudit.in \
-	seaudit-report-group.conf \
-        seaudit-report-service.conf \
-        seaudit-report-service.in
-
-seaudit-report-service: seaudit-report-service.in Makefile
-	sed -e 's|\@bindir\@|$(bindir)|g' $< > $@
-
-install-logwatch: $(dist_noinst_DATA) seaudit-report-service
-	mkdir -p -- $(LOGWATCH_GROUP)
-	install -m 644 seaudit-report-group.conf $(LOGWATCH_GROUP)
-	mkdir -p -- $(LOGWATCH_SERVICE)
-	install -m 644 seaudit-report-service.conf $(LOGWATCH_SERVICE)
-	mkdir -p -- $(LOGWATCH_FILTER)
-	install -m 755 seaudit-report-service $(LOGWATCH_FILTER)
+dist_noinst_DATA = dot_seaudit.in
 
 $(top_builddir)/libapol/src/libapol.so:
 	$(MAKE) -C $(top_builddir)/libapol/src $(notdir $@)
@@ -85,6 +62,4 @@ $(top_builddir)/libqpol/src/libqpol.so:
 $(top_builddir)/libsefs/src/libsefs.so:
 	$(MAKE) -C $(top_builddir)/libsefs/src $(notdir $@)
 
-.PHONY: install-logwatch
-
-CLEANFILES = dot_seaudit seaudit-report-service
+CLEANFILES = dot_seaudit
diff -up setools-3.3.7/secmds/Makefile.am.noship setools-3.3.7/secmds/Makefile.am
--- setools-3.3.7/secmds/Makefile.am.noship	2007-08-02 17:16:33.000000000 -0400
+++ setools-3.3.7/secmds/Makefile.am	2013-11-27 09:27:14.307196361 -0500
@@ -1,6 +1,6 @@
 # various setools command line tools
 
-bin_PROGRAMS = seinfo sesearch findcon replcon indexcon
+bin_PROGRAMS = seinfo sesearch findcon
 
 # These are for indexcon so that it is usable on machines without setools
 STATICLIBS = ../libsefs/src/libsefs.a ../libapol/src/libapol.a ../libqpol/src/libqpol.a -lsqlite3
diff -up setools-3.3.7/sediff/Makefile.am.noship setools-3.3.7/sediff/Makefile.am
--- setools-3.3.7/sediff/Makefile.am.noship	2007-04-25 15:20:20.000000000 -0400
+++ setools-3.3.7/sediff/Makefile.am	2013-11-27 09:27:14.307196361 -0500
@@ -1,13 +1,6 @@
 setoolsdir = @setoolsdir@
 
-dist_setools_DATA = sediff_help.txt sediffx.glade \
-	sediffx.png sediffx-small.png
-
-if BUILD_GUI
-  MAYBE_SEDIFFX = sediffx
-endif
-
-bin_PROGRAMS = sediff $(MAYBE_SEDIFFX)
+bin_PROGRAMS = sediff
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
 	@QPOL_CFLAGS@ @APOL_CFLAGS@ @POLDIFF_CFLAGS@
@@ -15,14 +8,7 @@ AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS
 
 LDADD = @SELINUX_LIB_FLAG@ @POLDIFF_LIB_FLAG@ @APOL_LIB_FLAG@ @QPOL_LIB_FLAG@
 
-sediff_CFLAGS = $(AM_CFLAGS)
-sediffx_CFLAGS = $(AM_CFLAGS) \
-	@GTK_CFLAGS@ @PIXBUF_CFLAGS@ @GLADE_CFLAGS@ @GTHREAD_CFLAGS@
-
-# need the -rdynamic flag below - glade uses dlopen() upon sediffx callbacks
-sediffx_LDFLAGS = $(AM_LDFLAGS) \
-	@GTK_LIBS@ @PIXBUF_LIBS@ @GLADE_LIBS@ @GTHREAD_LIBS@ @XML_LIBS@ \
-	-rdynamic
+sediff_CFLAGS = $(AM_CFLAGS) 
 
 DEPENDENCIES = $(top_builddir)/libpoldiff/src/libpoldiff.so \
 	$(top_builddir)/libapol/src/libapol.so \
@@ -30,20 +16,6 @@ DEPENDENCIES = $(top_builddir)/libpoldif
 
 sediff_SOURCES = sediff.c
 
-sediffx_SOURCES = \
-	find_dialog.c find_dialog.h \
-	open_policies_dialog.c open_policies_dialog.h \
-	policy_view.c policy_view.h \
-	progress.c progress.h \
-	remap_types_dialog.c remap_types_dialog.h \
-	result_item.c result_item.h \
-	result_item_render.c result_item_render.h \
-	results.c results.h \
-	select_diff_dialog.c select_diff_dialog.h \
-	toplevel.c toplevel.h \
-	utilgui.c utilgui.h \
-	sediffx.c sediffx.h
-
 $(top_builddir)/libpoldiff/src/libpoldiff.so:
 	$(MAKE) -C $(top_builddir)/libpoldiff/src $(notdir $@)
 
